analytic_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Analytic Operating Unit",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "ERP Harbor Consulting Services, "
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/operating-unit",
+    "category": "Sales",
+    "depends": ["analytic", "operating_unit"],
+    "data": [
```

Comment:
here `data` sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)